### ISSUE TITLE
Kohya_SS is making JSON instead of SAFETENSORS

### ISSUE THREAD

**Issue by KilluaMorales** [bmaltais/kohya_ss#3376]
Hey,

I just got into training loras. I got the basic steps on what to do, but every time I train a lora, it turns into JSON. I know I checked the box for SAFETENSORS, but every time I press the button, it keeps giving me JSON instead. Does anyone know where I'm going wrong?

**Comment by KilluaMorales** [comment]
02:10:16-143805 INFO     Start training LoRA Standard ...
02:10:16-145250 INFO     Validating lr scheduler arguments...
02:10:16-145250 INFO     Validating optimizer arguments...
02:10:16-146359 INFO     Validating C:/Users/djska/Downloads/Shizuno/log existence and writability... SUCCESS
02:10:16-146359 INFO     Validating C:/Users/djska/Downloads/Shizuno/model existence and writability... SUCCESS
02:10:16-147525 INFO     Validating stabilityai/stable-diffusion-xl-refiner-1.0 existence... SUCCESS
02:10:16-147525 INFO     Validating C:/Users/djska/Downloads/Shizuno/img existence... SUCCESS
02:10:16-149000 INFO     Validating E:/ComfyUI_windows_portable/ComfyUI/models/vae/sdxl_vae.safetensors existence... SUCCESS
02:10:16-159838 INFO     Regularization factor: 1
02:10:16-161193 INFO     Train batch size: 1
02:10:16-161193 INFO     Gradient accumulation steps: 1
02:10:16-162406 INFO     Epoch: 1
02:10:16-163414 INFO     Max train steps: 1600
02:10:16-163414 INFO     stop_text_encoder_training = 0
02:10:16-164463 INFO     lr_warmup_steps = 0
02:10:16-165545 INFO     Effective Learning Rate Configuration (based on GUI settings):
02:10:16-166549 INFO       - Main LR (for optimizer & fallback): 1.00e-04
02:10:16-167629 INFO       - Text Encoder (Primary/CLIP) Effective LR: 1.00e-03 (Specific Value)
02:10:16-168732 INFO       - Text Encoder (T5XXL, if applicable) Effective LR: 1.00e-03 (Inherited from Primary TE LR)
02:10:16-168732 INFO       - U-Net Effective LR: 1.00e-03 (Specific Value)
02:10:16-169734 INFO     Note: These LRs reflect the GUI's direct settings. Advanced options in sd-scripts (e.g., block LRs, LoRA+) can further modify rates
                         for specific layers.
02:10:16-171753 INFO     Saving training config to C:/Users/djska/Downloads/Shizuno/model\Shizuno Urushibara_20250808-021016.json...
02:10:16-174852 INFO     Executing command: C:\Users\djska\OneDrive\Desktop\kohya_ss\.venv\Scripts\accelerate.EXE launch --dynamo_backend no --dynamo_mode
                         default --mixed_precision fp16 --num_processes 1 --num_machines 1 --num_cpu_threads_per_process 2
                         C:/Users/djska/OneDrive/Desktop/kohya_ss/sd-scripts/sdxl_train_network.py --config_file
                         C:/Users/djska/Downloads/Shizuno/model/config_lora-20250808-021016.toml
ipex flag is deprecated, will be removed in Accelerate v1.10. From 2.7.0, PyTorch has all needed optimizations for Intel CPU and XPU.
2025-08-08 02:10:30 INFO     Loading settings from C:/Users/djska/Downloads/Shizuno/model/config_lora-20250808-021016.toml...             train_util.py:4651
C:\Users\djska\OneDrive\Desktop\kohya_ss\.venv\Lib\site-packages\transformers\tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884
  warnings.warn(
2025-08-08 02:10:31 INFO     Using DreamBooth method.                                                                                   train_network.py:517
                    INFO     prepare images.                                                                                              train_util.py:2072
                    INFO     0 train images with repeats.                                                                                 train_util.py:2116
                    INFO     0 reg images with repeats.                                                                                   train_util.py:2120
                    WARNING  no regularization images / 正則化画像が見つかりませんでした                                                  train_util.py:2125
                    INFO     [Dataset 0]                                                                                                  config_util.py:580
                               batch_size: 1
                               resolution: (1024, 1024)
                               resize_interpolation: None
                               enable_bucket: False


                    INFO     [Prepare dataset 0]                                                                                          config_util.py:592
                    INFO     loading image sizes.                                                                                          train_util.py:987
0it [00:00, ?it/s]
                    INFO     make buckets                                                                                                 train_util.py:1010
                    WARNING  min_bucket_reso and max_bucket_reso are ignored if bucket_no_upscale is set, because bucket reso is defined  train_util.py:1027
                             by image size automatically /
                             bucket_no_upscaleが指定された場合は、bucketの解像度は画像サイズから自動計算されるため、min_bucket_resoとmax_
                             bucket_resoは無視されます
                    INFO     number of images (including repeats) / 各bucketの画像枚数（繰り返し回数を含む）                              train_util.py:1056
                    INFO     mean ar error (without repeats): 0                                                                           train_util.py:1069
                    ERROR    No data found. Please verify arguments (train_data_dir must be the parent of folders with images) /        train_network.py:563
                             画像がありません。引数指定を確認してください（train_data_dirには画像があるフォルダではなく、画像があるフォ
                             ルダの親フォルダを指定する必要があります）
02:10:33-476788 INFO     Training has ended.

**Comment by Hirmuolio** [comment]
`INFO 0 train images with repeats. train_util.py:2116`

You don't have any training images.

In folder `C:/Users/djska/Downloads/Shizuno/img` you  should have another folder with name somthing like `1_images` and that folder should contain the images and their captions.

**Comment by KilluaMorales** [comment]
I have the images and their captions in that folder. So, I need to add another folder inside and transfer them into that folder?
